RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2355,9 +2355,22 @@ boolean L11_redZeppelin()
 	{
 		if(cloversAvailable() >= 3 && auto_shouldUseWishes())
 		{
-			makeGenieWish($effect[Fifty Ways to Bereave Your Lover]); // +100 sleaze dmg
-			makeGenieWish($effect[Dirty Pear]); // double sleaze dmg
-		}
+			if (auto_haveGenieBottle()) {
+				makeGenieWish($effect[Fifty Ways to Bereave Your Lover]); // +100 sleaze dmg
+				makeGenieWish($effect[Dirty Pear]); // double sleaze dmg
+			}
+			if (auto_haveMonkeyPaw())
+			{
+				if(have_effect($effect[Fifty Ways to Bereave Your Lover])==0)
+				{
+					auto_makeMonkeyPawWish($effect[Fifty Ways to Bereave Your Lover]);
```

Comment:
it might be good to calculate how much sleaze the user has -- while it is better to use the resource than not use the resource, in unrestricted paths it's pretty easy for high-shiny people to be at obscene sleaze levels where all they really need is dirty pear, not fifty + dirty pear. i would also probably invert the order here -- if the user has monkey paw, they should use those -before- using genie wishes, as genie wishes can be saved as 50k meat pocket wishes and an unused monkey paw wish is just kind of not valuable.

Reply:
I've added 400 as the amount of sleaze damage we consider "enough" and cast dirty pear before fifty ways.